NT-Line-team/Behave_test
# khanhvnn - Exercise Week 6: Develop test script for below functions of Opencart web application

## Exercise

* [Exercise-week6](https://github.com/NT-Line-team/Behave_test/issues/49)

## Overview

Develop test script for below functions of Opencart web application.
```
Search a product (finished)
```
## Verification Steps

1. `git checkout pr/###`
2. Check the functionality: 
- Run the files with below commands.
python ./search_product.py
(change the variables (e,g: port, search_phone.. where neccessary)
### Docs
